instagram-app/src/components/SearchBar/SearchBar.scss

```diff
@@ -0,0 +1,7 @@
+.show-sprite {
+  display: block;
+}
+
+.sprite {
```

Comment:
I really love the implementation of the sprite, it takes the application to a whole new level.